main.tf

```
@@ -21,7 +21,7 @@ module "self_signed_cert_ca" {
   source  = "cloudposse/ssm-tls-self-signed-cert/aws"
   version = "0.4.0"
 
-  name = "self-signed-cert-ca"
+  name = "${module.this.context.name}-self-signed-cert-ca"
```

Nice catch. All of these should probably be using attributes instead.

```suggestion
  attributes = ["self", "signed", "cert", "ca"]
```